include/mfmg/dealii/amge_host.templates.hpp

```diff
@@ -185,6 +186,34 @@ AMGe_host<dim, MeshEvaluator, VectorType>::compute_local_eigenvectors(
     solver.solve(agglomerate_system_matrix, agglomerate_mass_matrix,
                  inv_system_matrix, eigenvalues, eigenvectors);
   }
+  else if (_eigensolver_type == "lanczos")
+  {
+    boost::property_tree::ptree lanczos_params;
+#if 0
```

Comment:
If you really are going to keep these preprocessor directives, define something like LANCZOS_IS_DEFLATED and use it here.